docs/xkf/developer-guide/scheduling-scaling.md

```diff
@@ -180,7 +180,7 @@ antiAffinity rule: the pod should not be scheduled onto a node if that node is i
                 - key: prometheus
                   operator: In
                   values:
-                    - xks
+                    - xkf

```

Comment:
Should this match the config that we do in terraform-modules?
I think we use `xks` there.